service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java

```diff
@@ -155,19 +163,33 @@ private BeaconExportMaritimeUseDTO toMaritimeUse(BeaconUse use) {
     return BeaconExportMaritimeUseDTO
       .builder()
       .environment(use.getEnvironment().toString())
+      .typeOfUse(use.getUseType())
+      .beaconPosition(use.getBeaconPosition())
+      .beaconLocation(use.getBeaconLocation())
+      .windfarmLocation(use.getWindfarmLocation())
```

Comment:
Just checking do we need to update the unit tests for ExportMapper?

Reply:
Yeah that's next on my list